.github/workflows/testandvet.yml

```diff
@@ -23,29 +24,35 @@ jobs:
 
     steps:
 
-    - name: Checkout code
-      uses: actions/checkout@v4.2.2
-
-    - name: Install Go
-      uses: actions/setup-go@v5.3.0
-      with:
-        go-version-file: 'go.mod'
-
-    - name: Cache Go module and build cache
-      uses: actions/cache@v4.2.2
-      with:
-        key: go-${{ hashFiles('**/go.sum') }}
-        path: |
-          ~/go/pkg/mod
-        restore-keys: |
-          go-
-
-    - name: Install tennvet
-      run: |
-        GOBIN=$(pwd) go install github.com/tenntenn/tennvet@latest
-
-    - name: Test and vet
-      run: |
-        go vet ./...
-        go vet -vettool=$(pwd)/tennvet ./...
-        go test -v -race ./...
+      - name: Checkout code
```

Comment:
The go.mod specifies 1.23.7 as go directive but this PR chages for Go1.24 environment. Then the go.mod specifies 1.24.1 as toolchain directive.

Because `tennvet` checks whether use feature Go version build constraint, Install Go with toolchain directive version.